### ISSUE TITLE
Different versions of the same types of icons

### ISSUE THREAD

**Issue by afuno** [twbs/icons#346]
There is a problem that unites all such projects - this is the lack of variability. There are bold characters in the icons, there are subtle, there are ordinary. But as a rule, all these characters are the same size.

I do not mean all the icons, but only part of them. For example, a plus symbol. In my opinion, such characters deserve to have variability. They should, for example, be large in relation to (for example) the circle in which they are located. Or small.

We can customize the design. The same Bootstrap can be customized beyond recognition. But we cannot do this with icons. Instead, have to use several different sets with icons in projects. Because there is no variation.

To summarize, I wanted to offer to duplicate some icons (for example, circles, squares, etc.), but to resize the characters inside them.

**Comment by mdo** [comment]
This likely is a v2 thing for me.

Right now I’m creating consistent regular and fill versions of as many icons as possible. Eventually I may split this out into different groups (same repo) by style: thin, regular, bold, fill, duo tone, etc all come to mind. Unsure where it’ll lead, but fun to think about! :D
